presto-spi/src/main/java/com/facebook/presto/spi/PageBuilder.java

```diff
@@ -35,6 +35,16 @@
     private PageBuilderStatus pageBuilderStatus;
     private int declaredPositions;
 
+    /**
+     * Create a PageBuilder with given types.
+     * <p>
+     * PageBuilder constructed with this constructor has no estimation about bytes per entry.
+     * And it may have frequent resizing when appending rows.
```

Comment:
```
* A PageBuilder instance created with this constructor has no estimation about bytes per entry, 
* therefore it can resize frequently while appending new rows.
* <p>
* This constructor should only be used to get the initial PageBuilder.
* Once the PageBuilder is full use reset() or createPageBuilderLike() to create a new
* PageBuilder instance with its size estimated based on previous data.
```

Reply:
Please also update the commit message similarly to this javadoc.